src/content/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model.mdx

```diff
@@ -83,15 +83,17 @@ When [creating an access grant](#group-access), you can use our [standard roles]
 
 To view existing roles: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Administration**, then click **Organization and access**, and then click **Roles**.
 
-To create a custom role, click **Add new custom role**. Note that there are some hidden capabilities that are not available for adding to a custom role. For details, see [Capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#standard-roles). 
+To create a custom role, click **Add new custom role**. Peruse the list of available capabilities and decide which ones your custom role needs. 
 
-To learn more about how roles and groups relate, see [User management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+For more information about how roles and capabilities work, see [Capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#standard-roles). 
 
-## Add users [#add-users]
+Once you're finished creating a custom role, you will probably want to use it in a new [access grant](#group-access). 
 
-If you've created a new group, you must make sure that the users are added to that group. If users don't have the group assigned, no access will be granted. 
+## Add users to groups [#add-users]
 
-To view or manage users: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Administration**, and click **User management**. If you don’t see that option, review the [requirements](#requirements).
+In the user management UI, you can see what groups your users have been assigned. Your users only have access to the access grants associated with the groups they've been added to. 
```

Comment:
"you can see what groups your users have been assigned" --> "you can review your groups and the users assigned to them"